ipam/client/abstractipam.py

```diff
@@ -8,60 +8,60 @@ class AbstractIPAM:
 
     @abstractmethod
     def add_ip(self, ipaddr, dnsname, description):
-        pass
+        raise NotImplementedError
```

Comment:
Proper way would be to call constructor

``raise NotImplementedError()``

Reply:
Done